rpcs3/Emu/Cell/RawSPUThread.cpp

```diff
@@ -392,9 +392,30 @@ void spu_load_exec(const spu_exec_object& elf)
 
 	const auto funcs = spu->discover_functions(0, { spu->ls , SPU_LS_SIZE }, true, umax);
 
-	for (u32 addr : funcs)
+	if (spu_log.notice && !funcs.empty())
 	{
-		spu_log.success("Found SPU function at: 0x%08x", addr);
+		std::string to_log;
+		usz i = 0;
+
+		for (u32 addr : funcs)
+		{
+			if (i == 0 && funcs.size() < 8)
```

Comment:
This will remove all ', '. Move the i++ to the end of the function.

Reply:
Fixed.